x-pack/plugins/apm/public/components/app/TraceOverview/view.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiSpacer } from '@elastic/eui';
+import React from 'react';
+// @ts-ignore
+import { TraceListRequest } from '../../../store/reactReduxRequest/traceList';
+import EmptyMessage from '../../shared/EmptyMessage';
+import { TraceList } from './TraceList';
+
+interface Props {
+  urlParams: object;
+}
+
+export function TraceOverview(props: Props) {
+  const { urlParams } = props;
+
+  return (
+    <div>
+      <EuiSpacer />
+      <TraceListRequest
+        urlParams={urlParams}
+        render={({ data }: { data: object[] }) => (
```

Comment:
1. RRR should export types (adds mental TODO)
2. right now a custom selector can the structure of what 'render' returns. Perhaps render should always return:

```ts
interface ReactReduxRequestResponse<T> {
  data: T:
  status: 'LOADING' || 'SUCCESS' || 'FAILURE',
  args: any[]
}
```

... and then the return value from the selector becomes the value for `data`

Reply:
... Btw. do you prefer using RRR for data fetching and state, or just stateful components. I still can't decide what I like better.

Reply:
So far I'm good with the RRR stuff. I think I'd have to see how a stateful component with an HTTP request would look and work, I guess that would be without Redux?